app/controllers/api/v1/users/dashboard_controller.rb

```diff
@@ -4,6 +4,15 @@ def show
     render json: UserDashboardSerializer.new(fetch_dashboard_data(params[:id]))
   end
 
+  def destroy
+    user_wine = UserWine.find_by!(
+      user_id: params[:user_id],
+      wine_id: params[:wine_id]
+    ).destroy
+
+    render json: user_wine
```

Comment:
So just to confirm on this, when you destroy a wine, it displays the destroyed wine? Or would it display a `not_found` status?

Reply:
What gets rendered back is the 204 response for a user that returns an empty JSON body since the record has been destroyed